frontend/lib/src/components/core/Block/styled-components.ts

```diff
@@ -130,6 +131,19 @@ export const StyledColumn = styled.div<StyledColumnProps>(
       [`@media (max-width: ${theme.breakpoints.columns})`]: {
         minWidth: `calc(100% - ${theme.spacing.twoXL})`,
       },
+      ...(verticalAlignment === "bottom" && {
+        marginTop: "auto",
+        // Add margin to the last checkbox within the column to align it
+        // better with other input widgets. This is a temporary (ugly) fix
+        // until we have a better solution for this.
+        "& .element-container:last-of-type > .stCheckbox": {
+          marginBottom: 9,
```

Comment:
Should be "9px" right?

Reply:
Yep. But we can actually use 0.5rem here which is the exact value we need to correctly align this with 40px.